### PR TITLE
Update images in UI Overview

### PR DESCRIPTION
**Pull request by jfermi**
Updated images to 1.4.0 version of Monokle.  Added new settings and corrected grammar mistakes.

This PR...

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by olensmar — APPROVED** [review]
lgtm!

File: docs/overview.md [hunk 1]
```diff
@@ -1,43 +1,50 @@
...
-Monokle is layed out like many other desktop applications:
+Monokle is laid out like many other desktop applications:
 
-![Monokle UI](img/monokle-ui.png)
+![Monokle UI](img/monokle-ui-1.4.0.png)
 
 Left to right:
 
-- The vertical toolbar to the far left allows you to switch between File and Cluster mode.
+- The vertical toolbar to the far left allows you to switch between File, Kustomize, Helm and Cluster mode.
```
Comment (devcatalin) [inline docs/overview.md:9]: we should remove "and Cluster mode" from here since it was moved to the top header

Comment (olensmar) [reply docs/overview.md:9]: nice catch @devcatalin

**Comment by jfermi** [comment]
I still see the cluster icon in the left toolbar under the Helm icon.  Am I
looking at an old version?

**Comment by jfermi** [comment]
I see from this morning that 1.5.0 has this change.  I guess this begs the
question - for the work I am doing right now, should I go ahead and work
with 1.5.0 since the release is imminent?

**Comment by olensmar** [comment]
yeah - that's what I was thinking also - perhaps we just make minor high-level changes to the existing 1.4.0 documentation and then double-down on the upcoming 1.5.0...

File: docs/overview.md (hunk 1 above)
Comment (olensmar) [reply docs/overview.md:9]: or actually - this is for 1.4.0 - so it's ok for now... will have to change in the 1.5.0 documentation

**Pull request merged**